portal-api/src/v1/gef/models/facilities.js

```diff
@@ -163,6 +164,10 @@ class Facility {
         this.monthsOfCover = null;
       }
 
+      if (req.coverDateConfirmed) {
+        this.coverDateConfirmed = req.coverDateConfirmed;
+      }
+
```

Comment:
Can we update the "update facility" portal api test to test these new conditions/additions